templates/download/server/choose.html

```diff
@@ -69,6 +72,40 @@ <h4 class="p-muted-heading">Explore:</h4>
       </ul>
     </div>
   </div>
+  {% if releases.latest.short_version > releases.lts.short_version %}
+  <div class="p-strip is-shallow">
+    <div class="u-fixed-width">
+      <hr>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-6 p-divider__block">
+      <h2>Ubuntu Server {{ releases.latest.full_version }}</h2>
+      <p class="u-no-padding--top">The latest version of Ubuntu Server, including nine months of security and maintenance updates, until {{ releases.latest.eol }}.</p>
```

Comment:
This padding is for vertical alignment nudging. Do you feel the space is too large between the elements?